dataflow/gpu-examples/tensorflow-landsat/main.py

```diff
@@ -290,45 +297,54 @@ def run(
         "--output-path-prefix",
         required=True,
         help="Path prefix for output image files. "
-        "This can be a Google Cloud Storage path.",
-    )
+        "This can be a Google Cloud Storage path.")
     parser.add_argument(
         "--scene",
         dest="scenes",
         action="append",
         help="One or more Landsat scene IDs to process, for example "
         "LC08_L1TP_109078_20200411_20200422_01_T1. "
         "They must be in the format: "
-        "https://www.usgs.gov/faqs/what-naming-convention-landsat-collections-level-1-scenes",
-    )
+        "https://www.usgs.gov/faqs/what-naming-convention-landsat-collections-level-1-scenes")
     parser.add_argument(
         "--rgb-band-names",
         nargs=3,
         default=DEFAULT_RGB_BAND_NAMES,
-        help="List of three band names to be mapped to the RGB channels.",
-    )
+        help="List of three band names to be mapped to the RGB channels.")
     parser.add_argument(
         "--min",
         type=float,
         default=DEFAULT_MIN_BAND_VALUE,
-        help="Minimum value of the band value range.",
-    )
+        help="Minimum value of the band value range.")
     parser.add_argument(
         "--max",
         type=float,
         default=DEFAULT_MAX_BAND_VALUE,
-        help="Maximum value of the band value range.",
-    )
+        help="Maximum value of the band value range.")
     parser.add_argument(
-        "--gamma", type=float, default=DEFAULT_GAMMA, help="Gamma correction factor."
-    )
+        "--gamma",
+        type=float,
+        default=DEFAULT_GAMMA,
+        help="Gamma correction factor.")
+    parser.add_argument(
+        "--gpu-type",
+        default=DEFAULT_GPU_TYPE,
+        help="GPU type hint to use.")
```

Comment:
technically, we have one resource hint (--accelerator), that captures both type and count. you could still customize type and count separately if you prefer, but perhaps don't call it a 'hint' to avoid confusion.

Reply:
I would prefer to capture them separately to avoid user errors as much as possible. That way the accelerator hint format can stay as an implementation detail. I'll remove the word "hint".